src/android/device_watcher.cpp

```diff
@@ -3,10 +3,7 @@
 #include "device_watcher.h"
 #include <vector>
 #include <sstream>
-#include <jni.h>
```

Comment:
Why was this removed?
No impact on android? (we do not build it in our CI)

Reply:
We build it in GHA, which is where I saw things fail until I fixed these include.
I don't remember why `jni.h` is needed or why I removed, but it compiled without it so there doesn't seem to be a dependency in this .cpp...